shapash/webapp/smart_app.py

```diff
@@ -154,7 +159,17 @@ def init_data(self):
 
         self.dataframe['_index_'] = self.explainer.x_init.index
         self.dataframe.rename(columns={f'{self.predict_col}': '_predict_'}, inplace=True)
-        col_order = ['_index_', '_predict_'] + self.dataframe.columns.drop(['_index_', '_predict_']).tolist()
+        if self.explainer.y_target is not None:
+            self.dataframe = self.dataframe.join(
+                self.explainer.y_target.rename(columns={self.explainer.y_target.columns[0]:"_target_"}), 
+            )
+            if self.explainer._case == 'regression':
```

Comment:
maybe, it would be better to compute error in smart_explainer, and use attribute in this app and in the prediction_picking plot